statsmodels/nonparametric/kernel_regression.py

```diff
@@ -767,9 +767,9 @@ def _compute_se_lambda(self, Y, X):
         but slows down computation significantly.
         """
         n = np.shape(Y)[0]
-        lam = np.empty(shape=(self.nres, ))
+        lam = np.empty(shape=(self.nres,))
         for i in range(self.nres):
-            ind = np.random.random_integers(0, n-1, size=(n,1))
+            ind = np.random.randint(0, n-1, size=(n, 1))
```

Comment:
Needs to be (0,n,size=(n,1)) -> not the deprecation warning saying +1. Former was closed.  This is open.